### DISCUSSION TITLE
Codespaces not logging to a Microsoft Account for Sync Settings

### DISCUSSION THREAD

**Discussion by valfrid-ly** [community/community#5954] [Projects and Issues]
When trying to sign in with a Microsoft account in Sync Settings it fails to log in with message "Error while turning on Settings Sync. Unable to login"

VSCode Version: 1.60.2
Local OS Version: 1.60.2
Reproduces in: Codespaces
Name of Dev Container Definition with Issue: tmsmith/Dapper-Extensions
Steps to Reproduce:

1. Setup both accounts on your local VSCode, github and microsoft
2. Open github.dev
3. In accounts button, sign in (if not signed yet) to your github account
4. In accounts button click on "Turn on Settings Sync..."
5. Click in "Sign in & Turn on" button
6. Select "Sign in with Microsoft"
7. Log in to your Microsoft account in the new opened tab
8. Error while turning on Settings Sync. Unable to login should appear on codespaces tab in the bottom right corner

**Comment by joseantoniocb** [comment]
Hello @valfrid-ly I'm having a similar issue, when I click on the settings sync button nothing happens.
I'm able to login and sync my setting successfully on my Mac native app and with any browser, even on my iPad (http://vscode.dev)  but this issue is only present when I use http://github.dev web based editor.

**Comment by diminutivesloop** [comment]
I [reported this issue already](https://github.com/microsoft/vscode/issues/170717) on the VSCode repo. I use github.dev for reviewing PRs but I prefer to use my personal Microsoft account for settings sync since we use work-specific GitHub accounts and I want to be able to sync my VSCode settings between my work and personal machines. My use-case would also be solved if VSCode supported signing-in with more than one GitHub account at once.
